Sources/PostgreSQL/Message/PostgreSQLMessage.swift

```diff
@@ -2,6 +2,11 @@ import Bits
 
 /// A frontend or backend PostgreSQL message.
 enum PostgreSQLMessage {
+    /// Identifies the message as SSL opening sequence
+    case sslMessage(PostgreSQLSSLMessage)
+    /// Identifies the message as the response from the server to SSL message
+    case sslRequest(PostgreSQLSSLRequest)
```

Comment:
Wouldn't that be a `sslResponse` then?

Reply:
In my opinion, I would name them sslRequest and sslResponse